README.md

```diff
@@ -1,2 +1,60 @@
+[![CircleCI](https: circleci.com/gh/giantswarm/appresource/tree/master.svg?style=shield)](https: circleci.com/gh/giantswarm/appresource/tree/master)
+
 # appresource
```

Comment:
I really dislike the idea of having one repo per resource. Is this for a reason?

Reply:
We started our conversation in here, settled to set up the different repo. https://github.com/giantswarm/cluster-operator/pull/613#discussion_r319177680

Reply:
I also think it's too much, Next time when we have another resource, then we should create another repo like this?

Reply:
When there is another resource that fits in here we can add it and do stuff it is only that we only had one resource for now and `apiextensions` is totally the wrong place for this IMO. `operatorkit` is the wrong place because of dependency graph issues so its own repo was the way to go.